implementation/helper/boost/asio/detail/impl/socket_ops_ext_local.ipp

```diff
@@ -122,13 +166,25 @@ signed_size_type recvfrom(socket_type s, buf* bufs, size_t count,
 
   union {
     struct cmsghdr cmh;
-    char   control[CMSG_SPACE(sizeof(struct ucred))];
+#ifdef __QNX__
+    char   control[512];
+#else
+    char   control[CMSG_SPACE(sizeof(UCRED_T))];
```

Comment:
Another usage of UCRED_T outside of __QNX__ ifdef.

Reply:
Fixed.